### ISSUE TITLE
Upcoming changes to rendering of URLs in quoted tweets

### ISSUE THREAD

**Issue by glciampaglia** [IUNetSci/hoaxy-botometer#258]
Twitter has [recently announced](https://twittercommunity.com/t/updating-how-urls-are-rendered-in-the-quote-tweet-payload/105473) that there will be a few changes to how URLs are rendered in quote tweets. Is this going to affect in any way Hoaxy?

**Comment by mihaivavram** [comment]
@glciampaglia May want to reflect this in the other issues you have posted in the different repos/projects:

```
Note that there will be a 30-day period when Quote Tweets JSON will reflect both formats: the new "quoted_status_permalink" attributes will be provided, along with the older style of appending a t.co URL to the quoted Tweet message and putting quote Tweet URL metadata in the root-level "entities" object. After approximately June 15, 2018, only the updated Quote Tweet format will be provided. Be sure to update your parsers by then!
```
Reference: [Twitter Docs](https://developer.twitter.com/en/docs/tweets/data-dictionary/overview/intro-to-tweet-json#quotetweet)

**Comment by mihaivavram** [comment]
Update for Hoaxy-Botometer: 

For Quote Tweets we only consider the top-most layer of the Network, meaning A tweeted B regardless of mention, retweet, or quote.  Hence, we only consider the tweet of A which then references B in some way. Twitter's change alters the way the B tweet can be retrieved for quote tweets, but our Twitter/Hoaxy search does not use that information. Our Twitter/Hoaxy search only uses the tweet of A computed via `https://twitter.com/<user_A_id>/status/<A_tweet_id>`.

**TLDR; This new Twitter change should not affect the Hoaxy-Botometer front-end.**

**Comment by glciampaglia** [comment]
Thank you! Closing then.
